Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:alpine
```

Comment:
Btw: golang is not needed to run feed2imap-go -- only to compile it. Simple alpine (or even less...) should be enough